plugins/repository-azure/src/internalClusterTest/java/org/opensearch/repositories/azure/AzureBlobStoreRepositoryTests.java

```diff
@@ -188,6 +190,7 @@ protected String requestUniqueId(final HttpExchange exchange) {
     @SuppressForbidden(reason = "this test uses a HttpServer to emulate an Azure endpoint")
     private static class AzureHTTPStatsCollectorHandler extends HttpStatsCollectorHandler {
 
+        private final Logger testLogger = LogManager.getLogger("AzureStatsLogger");
```

Comment:
Can be static I think. Also, I'd suggest adding "Test" to the name somewhere so it is clear in the output this is only a test logger (you could also maybe just stick with the convention of using the class then it is clear that a test class is the thing doing the logging).

Reply:
Makes sense. I'll stick to the class name. Thanks @andrross !!